### DISCUSSION TITLE
After deploy , needs to links GPOs to OU (Ok normal) but which ones is recommanded ?

### DISCUSSION THREAD

**Discussion by captainmac40** [LoicVeirman/HardenAD#140] [Q&A] [answered]
Hello, first of all, amazing work!

I deployed Harden on my new AD after mounting it with HelloMyDir (great tool too) and put my PAW in the PAWAccess OU.
I thought at first that the GPO linked to this type of workstation was automatically linked to this OU but it is not the case...

So, which one is recommended to apply to a PAW? All would be an easy answer... ^^

Thanks a lot!

PS : Should I suggest to add to documentation this prerequires : 
- User used to apply Harden should be Schema admin (for laps) --- Edit : RTFM it's in ....
- .NET Framework installed (again for LAPS)
Or maybe it's linked to my config ... :)

**Comment by captainmac40** [comment]
Reply to myself : GPO was linked I re-read script and debug log ... :)

GPO range seems to be to global DC not particular OU.

In French because my English wasn't very good ... : Je pensais que les GPO recommandés pour tel ou tel type de machine était liées à l'OU en question (je prends l'exemple des PAW). A ce que je vois sur les logs il y a bien des liens avec les OU mais j'ai pas du bien regarder dans l'AD ....

Si quelqu'un peu me confirmer le truc :)

Merci !

**Comment by LoicVeirman** [comment]
J’ai ren compris 😊

Le 29/11/2024 14:48, « captainmac40 » ***@***.***> a écrit :


CAUTION: This email originated from outside your organization. Exercise caution when opening attachments or clicking links, especially from unknown senders.

Reply to myself : GPO was linked I re-read script and debug log ... :)

GPO range seems to be to global DC not particular OU.

In French because my English wasn't very good ... : Je pensais que les GPO recommandés pour tel ou tel type de machine était liées à l'OU en question (je prends l'exemple des PAW). A ce que je vois sur les logs il y a bien des liens avec les OU mais j'ai pas du bien regarder dans l'AD ....

Si quelqu'un peu me confirmer le truc :)

Merci !

**Reply by captainmac40** [reply]
Je reformule : Je pensais qu'en mettant mon PAW-T0 dans l'OU PawAccess ou PAW-T0 j'allais avoir toutes les GPO recommandées pour ce genre de machine qui allaient s'appliquer ""toute seules"" (je pense par exemple aux GPO pour Bitlocker ou d'autres en lien avec la sécurisation du PAW. J'ai surement pas mis ma machine dans le bon OU ... :)

Merci en tout cas et super boulot, juste une évolution peut-être : le préfixe des OU qu'on puisse donner en début de script comme le domaine par exemple pour pas avoir à modifier à la main le tasksequence :)

**Comment by LoicVeirman** [comment]
La 2.10 est plus simple de ce côté et va ouvrir l’accès à une « customisation » plus dynamique. Mais ce sera pour la 3.0…
(ou alors, on me demande une presta et je vous fait un code sur mesure rien que pour vous)

Le 29/11/2024 15:08, « captainmac40 » ***@***.***> a écrit :


CAUTION: This email originated from outside your organization. Exercise caution when opening attachments or clicking links, especially from unknown senders.

Je reformule : Je pensais qu'en mettant mon PAW-T0 dans l'OU PawAccess ou PAW-T0 j'allais avoir toutes les GPO recommandées pour ce genre de machine qui allaient s'appliquer ""toute seules"" (je pense par exemple aux GPO pour Bitlocker ou d'autres en lien avec la sécurisation du PAW. J'ai surement pas mis ma machine dans le bon OU ... :)

Merci en tout cas et super boulot, juste une évolution peut-être : le préfixe des OU qu'on puisse donner en début de script comme le domaine par exemple pour pas avoir à modifier à la main le tasksequence :)

**Reply by captainmac40** [reply]
Cool bonne nouvelle, j'ai modifié le tasksequence à la mano ça s'est bien passé :)
Je vais tester en déplaçant mon PAW dans une autre OU pour voir ce que donne le GPResult :)
Encore bravo en tout cas à toute la team, super outil pour mettre en place du tiering facilement (sur un DC tout neuf c'est plus facile ^^), et en passant avant HelloMyDir je pense que mon DC doit être pas trop mal au niveau config ^^

**Comment by LoicVeirman** [comment, marked as answer]
Ah bah c’est le but du produit ^^

Le 29/11/2024 15:15, « captainmac40 » ***@***.***> a écrit :


CAUTION: This email originated from outside your organization. Exercise caution when opening attachments or clicking links, especially from unknown senders.

Cool bonne nouvelle, j'ai modifié le tasksequence à la mano ça s'est bien passé :)
Je vais tester en déplaçant mon PAW dans une autre OU pour voir ce que donne le GPResult :)
Encore bravo en tout cas à toute la team, super outil pour mettre en place du tiering facilement (sur un DC tout neuf c'est plus facile ^^), et en passant avant HelloMyDir je pense que mon DC doit être pas trop mal au niveau config ^^
